Register quantized ops into quantization example

larryliu0820 opened the pull request:
Summary:
Enables the ability to `export_to_pte`. Previously quantized models can't run `export_to_pte` because some quantized ops are missing out variants. Recently we added support for custom ops and register them into EXIR by loading shared library.

This diff adds support for registering quantized ops out variants and add it into CI.

Differential Revision: D48541611

jerryzh168 reviewed (approved):
thanks! does it work for per channel weight quant as well? (changing is_per_channel to True instead of False in the quant example)

larryliu0820 committed:
Register quantized ops into quantization example (#85)

Summary:
Pull Request resolved: https://github.com/pytorch/executorch/pull/85

Enables the ability to `export_to_pte`. Previously quantized models can't run `export_to_pte` because some quantized ops are missing out variants. Recently we added support for custom ops and register them into EXIR by loading shared library.

This diff adds support for registering quantized ops out variants and add it into CI.

Reviewed By: huydhn

Differential Revision: D48541611

fbshipit-source-id: 113b9af4bfc3b47673453e6f898d2205f361a879
